### DISCUSSION TITLE
minisector value

### DISCUSSION THREAD

**Discussion by thefeli73** [br-g/openf1#77] [General]
the docs mention these minisector values:
0 	not available
2048 	yellow
2049 	green
2050 	?
2051 	purple
2052 	?
2064 	?
2068 	?

value 2064 seems to indicate the pitlane, i suggest that info is added to the docs

**Comment by br-g** [comment]
Added!
Thanks a lot Felix, let me know if you find the meaning of 2050, 2052 and 2068 :wink:

**Comment by mohamedhadrami** [comment]
This is what I have so far, 

export const segmentColor: Record<number, string> = {
    0: '#52525B',
    2048: '#FFFF00', // No improvement on PB (Yellow)
    2049: '#00FF00', // PB (Green)
    2050: 'white', // Unknown
    2051: '#FF00FF', // Best (Purple)
    2052: 'white', // Unknown
    2064: '#0000FF', // Enter/ exit boxes (Blue)
    2068: 'white' // Unknown
};

Would be great to figure out what the others could mean
